app/controllers/applications_controller.rb

```diff
@@ -0,0 +1,39 @@
+class ApplicationsController < ApplicationController
+  def show 
+    @app = Application.find(params[:id]) 
+    @pets = @app.pets
+    if params[:search]
+      @found_pets = Pet.search(params[:search])
+    end
+  end 
+
+  def new; end
+
+  def create
+    new_app = Application.new(strong_params)
+    if new_app.save
+      redirect_to "/applications/#{new_app.id}"
+    else 
+      # require 'pry'; binding.pry
+      flash[:notice] = new_app.errors.full_messages.to_sentence
+      redirect_to "/applications/new"
+    end 
+  end
+
+  def update
+    app = Application.find(params[:id]) 
+    app.update!(description: params[:description], status: 1)
+    redirect_to "/applications/#{app.id}"
+  end
```

Comment:
I suggest not using `!` methods in the controller and instead using an if/else using .save like you did in your create method. If for whatever reason line 25 didn't update, your app would result in an error page instead of handling it gracefully using a flash message or some other means.